dotnet/src/Microsoft.AutoGen/Abstractions/IAgentRuntime.cs

```diff
@@ -13,6 +13,7 @@ public interface IAgentRuntime
     ValueTask<AgentState> ReadAsync(AgentId agentId, CancellationToken cancellationToken = default);
     ValueTask SendResponseAsync(RpcRequest request, RpcResponse response, CancellationToken cancellationToken = default);
     ValueTask SendRequestAsync(IAgentBase agent, RpcRequest request, CancellationToken cancellationToken = default);
+    ValueTask SendMessageAsync(Message message, CancellationToken cancellationToken = default);
```

Comment:
Not related to this PR, but I wonder how close we are to combine `IAgentRuntime` and `IAgentWorker`, they looks so similar

![image](https://github.com/user-attachments/assets/501c101d-8727-4b75-8887-0a9c89dcca84)